recipes/podofo/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class OatppSwaggerConan(ConanFile):
+    name = "podofo"
+    license = "GPL-3.0", "LGPL-3.0"
+    homepage = "http://podofo.sourceforge.net"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "PoDoFo is a library to work with the PDF file format."
+    topics = ("conan", "PDF", "PoDoFo", "podofo")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake", "cmake_find_package"
+    exports_sources = "CMakeLists.txt"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+        if self.settings.os == "Macos" and self.options.shared:
+            raise ConanInvalidConfiguration("currently this recipe doesn't support shared libraries on MacOS")
+
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "5":
+            raise ConanInvalidConfiguration("PoDoFo requires GCC >=5")
+
+    def requirements(self):
+        self.requires("freetype/2.10.4")
+        if str(self.settings.os) in ["Linux", "Macos"]:
+            self.requires("fontconfig/2.13.92")
+        self.requires("libjpeg/9d")
+        self.requires("libunistring/0.9.10")
+        self.requires("libtiff/4.1.0")
+        self.requires("libidn/1.36")
+        self.requires("openssl/1.1.1h")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("podofo-{0}".format(self.version), self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+        self._cmake.definitions["PODOFO_BUILD_LIB_ONLY"] = True
+        self._cmake.definitions["PODOFO_BUILD_SHARED"] = self.options.shared
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
```

Comment:
Seems to be unecessary, or maybe for some triplets I've missed?

```
----Running------
> cmake --build '/home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/build/297a3685555a44e7f85312cead1d43f33ce5eb15/build_subfolder' '--target' 'install' '--' '-j3'
-----------------
[100%] Built target podofo_static
Install the project...
-- Install configuration: "Debug"
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/podofo-base.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/podofo.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/podofo_config.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/Pdf3rdPtyForwardDecl.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfArray.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfCanvas.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfColor.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfCompilerCompat.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfCompilerCompatPrivate.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfContentsTokenizer.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfData.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfDataType.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfDate.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfDefines.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfDefinesPrivate.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfDictionary.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfEncoding.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfEncodingFactory.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfEncrypt.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfExtension.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfError.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfFileStream.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfFilter.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfFiltersPrivate.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfImmediateWriter.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfInputDevice.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfInputStream.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfLocale.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfMemStream.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfMemoryManagement.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfName.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfObject.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfObjectStreamParserObject.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfOutputDevice.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfOutputStream.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfParser.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfParserObject.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfRect.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfRefCountedBuffer.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfRefCountedInputDevice.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfReference.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfStream.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfString.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfTokenizer.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfVariant.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfVecObjects.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfVersion.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfWriter.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfXRef.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfXRefStream.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/PdfXRefStreamParserObject.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/podofoapi.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/util/PdfMutex.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/util/PdfMutexImpl_noop.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/util/PdfMutexImpl_win32.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/util/PdfMutexImpl_pthread.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/base/util/PdfMutexWrapper.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfAcroForm.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfAction.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfAnnotation.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfCMapEncoding.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfContents.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfDestination.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfDifferenceEncoding.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfDocument.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfElement.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfEncodingObjectFactory.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfExtGState.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfField.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFileSpec.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFont.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontCID.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontCache.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontConfigWrapper.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontFactory.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontFactoryBase14Data.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontMetrics.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontMetricsBase14.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontMetricsFreetype.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontMetricsObject.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontSimple.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontTTFSubset.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontTrueType.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontType1.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontType3.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFontType1Base14.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfFunction.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfHintStream.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfIdentityEncoding.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfImage.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfInfo.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfMemDocument.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfNamesTree.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfOutlines.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfPage.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfPagesTree.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfPagesTreeCache.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfPainter.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfPainterMM.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfShadingPattern.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfSignOutputDevice.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfSignatureField.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfStreamedDocument.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfTable.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfTilingPattern.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/include/podofo/doc/PdfXObject.h
-- Installing: /home/conan/w/cci_PR-3594/.conan/data/podofo/0.9.6/_/_/package/297a3685555a44e7f85312cead1d43f33ce5eb15/lib/libpodofo.a
[HOOK - conan-center.py] post_package(): [PACKAGE LICENSE (KB-H012)] OK
[HOOK - conan-center.py] post_package(): [DEFAULT PACKAGE LAYOUT (KB-H013)] OK
[HOOK - conan-center.py] post_package(): [MATCHING CONFIGURATION (KB-H014)] OK
[HOOK - conan-center.py] post_package(): [SHARED ARTIFACTS (KB-H015)] OK
[HOOK - conan-center.py] post_package(): [PC-FILES (KB-H020)] OK
[HOOK - conan-center.py] post_package(): [CMAKE-MODULES-CONFIG-FILES (KB-H016)] OK
[HOOK - conan-center.py] post_package(): [PDB FILES NOT ALLOWED (KB-H017)] OK
[HOOK - conan-center.py] post_package(): [LIBTOOL FILES PRESENCE (KB-H018)] OK
[HOOK - conan-center.py] post_package(): [MS RUNTIME FILES (KB-H021)] OK
```

Reply:
This was suggested pr https://github.com/conan-io/conan-center-index/pull/3594/commits/32f6a22fbcd17a0ad960f7586f836106284a03b1#r525908108

Reply:
Interesting, pkg-config file seems to be installed only if shared...

Therefore, seems that `tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))` is useless.

Reply:
Ok, I'll remove it